elixir/apps/domain/lib/domain/billing/stripe/processed_events/processed_event.ex

```diff
@@ -0,0 +1,19 @@
+defmodule Domain.Billing.Stripe.ProcessedEvents.ProcessedEvent do
+  @moduledoc """
+  Schema for tracking processed Stripe webhook events.
+  Ensures idempotency and chronological ordering of event processing.
+  """
+
+  use Domain, :schema
+
+  @primary_key {:stripe_event_id, :string, []}
+  schema "processed_stripe_events" do
+    field :event_type, :string
+    field :processed_at, :utc_datetime_usec
+    field :stripe_customer_id, :string
+    field :event_created_at, :utc_datetime
```

Comment:
Just double-checking, this is not `usec` because we don't have that precision from Stripe right?

Reply:
Correct, it give us a Unix timestamp in seconds.

From the docs:
```
Time at which the object was created. Measured in seconds since the Unix epoch.
```